core/src/test/java/tech/tablesaw/SortTest.java

```diff
@@ -149,4 +151,20 @@ public void createSortInvalidPrefixColumnDoesNotExist() {
 
     assertEquals("Unrecognized Column: '>col2'", thrown.getMessage());
   }
+
+  @Test
+  public void testGetOrderAndCreate0() throws Exception {
```

Comment:
this test name should be more descriptive. what behavior is being tested? i.e. why do you expect an `IllegalStateException` to be thrown?